pages/Product/components/Reviews/components/Header/components/RatingCount/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import styles from './style';
+
+/**
+ * Rating count
+ * @param {number} count The number of ratings given for a product
+ * @returns {null|JSX}
+ */
+const RatingCount = ({ count }) => {
+  if (!count) {
+    return null;
+  }
+
+  return (
+    <I18n.Text
```

Comment:
Please make this single line, there should be enough room.

Reply:
Done.